ui/src/js/profile/ReceiptsPage.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import {Link} from 'react-router';
+import Boostrap from 'bootstrap';
+
+import Receipts from './tools/Receipts';
+
+require("./receipts.less");
+
+export default class Tools extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      view: 'receipts'
+    };
+  }
+
+  switchTo(value) {
+    this.setState({view: value})
+  }
+
+  render() {
+    var content;
```

Comment:
Please set a default value to clarify what we expect in of no receipts. Thanks.

Reply:
Added default contnt